Back/Reciplastk.Gateway/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Reciplastk.Gateway.DataAccess;
+using Reciplastk.Gateway.Models;
+using Reciplastk.Gateway.Services;
+
+namespace Reciplastk.Gateway.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProductsController : ControllerBase
+    {
+       
+        private readonly ProductsService productsService;
+
+        public ProductsController( ProductsService productsService)
+        {
+            this.productsService = productsService;
+        }
+
+        [HttpGet("getProducts")]
```

Comment:
This can be renamed to "GetAll"
```suggestion
        [HttpGet("GetAll")]
```

Reply:
Se unificaron los nombre de los metodos